omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1299,6 +1299,8 @@ def draw_labels(self, panel, page):
                             label_value.append(channel["label"])
                     label_value = " ".join(label_value)
 
+                elif  prop_nf[0] in ["zoom"]:
```

Comment:
The build is failing flake8 checks due to this line
```
flake8.main.application   MainProcess   1787 INFO     Found a total of 79 violations and reported 1
./omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py:1302:21: E271 multiple spaces after keyword
```

Reply:
Thanks, I suppressed the extra space, it should be ok now